resources/js/app.js

```diff
@@ -19,8 +19,15 @@
    otherwise({
     redirectTo: '/home'
    });
+       $locationProvider.html5Mode({
```

Comment:
@hemangsk can we check indentation with a coafile? @abhsag24 is indentationbear good enough for this btw.?